scylla-persistence/src/main/java/com/netflix/conductor/scylla/dao/ScyllaExecutionDAO.java

```diff
@@ -1204,8 +1215,17 @@ String lookupShardIdFromWorkflowId(String workflowId) {
         }
     }
 
+    public boolean isConcurrencyLimitEnabled() {
+        return isConcurrencyLimitEnabled;
+    }
+
+    public void setConcurrencyLimitEnabled(boolean concurrencyLimitEnabled) {
+        this.isConcurrencyLimitEnabled = concurrencyLimitEnabled;
+    }
+
     @Override
     public void setApplicationContext(ApplicationContext applicationContext) throws BeansException {
         this.redisLock = (RedisLock) applicationContext.getBean("provideRedisLock");
+        setConcurrencyLimitEnabled(Boolean.parseBoolean(applicationContext.getEnvironment().getProperty("concurrencyLimitEnabled")));
```

Comment:
to avoid  application failure we can add default also 
applicationContext.getEnvironment().getProperty("concurrencyLimitEnabled", "false")